test/test_mitm_initialization.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+
+#
+# This file is part of the PyRDP project.
+# Copyright (C) 2020 GoSecure Inc.
+# Licensed under the GPLv3 or later.
+#
+
+"""
+Integration test for the initialization of pyrdp-mitm.py.
+It could be enhanced with relevant assertions but for now only executes the code to catch
+potential basic errors/import problems.
+"""
+from pyrdp.mitm.cli import configure
+
+
+if __name__ == "__main__":
+    configure()
```

Comment:
Importing something that is already an import in the `bin/` front-end seems stretched to me. Might as well go straight to core but I see your reasoning you want to exercise the code in `bin/pyrdp-mitm.py`.

Would this catch the bug you were trying to catch earlier?

Reply:
I'm not sure what you mean by "might as well go straight to core"...

But damnit I went way too fast and no what I did would not have caught the bug I introduced, because I forgot to actually initialize the MITM. I pushed changes and it should be good now (will confirm in a min)

Reply:
alright now its good thanks for the catch 😬